config/lapack.py

```diff
@@ -580,7 +556,7 @@ def blas_complex_return():
     if (rc == 0):
         config.environ.append( 'CXXFLAGS', define('COMPLEX_RETURN_ARGUMENT') )
     else:
-        print_warn( 'unexpected error!' )
+        raise Error( "Could not determine how zdot returns complex result; check log." )
```

Comment:
Gave it a better error message and made it a hard error.

Reply:
OK